english/captain/captain.md

```diff
@@ -1,5 +1,5 @@
 Good Choice.
 
 A man of honour beats a genius billionaire playboy philanthropist.
-(But can he do it again, later?)
+(But can he do it again, later?) or vote for the cool [guy](../ironman/ironman.md)
 Let's watch the trailer of him fighting iron man [here](https://youtu.be/uVdV-lxRPFo).
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.